### DISCUSSION TITLE
Request :- Custom Build Options.

### DISCUSSION THREAD

**Discussion by atikattar1104** [Sketchware-Pro/Sketchware-Pro#647] [Q&A] [answered]
Custom Build Options To Release '.apk', '.apks', '.xapk', '.aab', '.apk+.obb', Etc. Files Rather Than Only Direct App Install On Device. Maybe Helpful When Sharing Created App To Someone.

**Comment by atikattar1104** [comment]
'.apk+.obb', '.xapk', '.apks', And Other Types Are Useful When Creating Apps With Extra Heavy Resources (Like Audio Files For Launchpad App, And Big Videos Which You Want To Embed In Your App, And Etc.).

**Comment by atikattar1104** [comment]
Didn't Knew Some Of These Features Are Available As 'Export' Option, But I Think Other Options May Help.

**Comment by j-px** [comment, marked as answer]
There's the Sign/Export menu for release mode .aab and .apk files. More options like .xapk or .apks aren't planned, as you can convert e.g. a .aab file to a .apks file using a specialized conversion tool.

OBB files aren't intended for use by apps that are commonly created with Sketchware Pro. They should be used when assets like lots of game map textures or binaries are included with the app, and having them in the APK would make it larger than 100 MB.  
More on OBB files here: https://developer.android.com/google/play/expansion-files
